main_loop_tf/validate.py

```diff
@@ -121,14 +145,20 @@ def validate(placeholders,
             # Get batch pred, mIoU so far, batch loss and potentially the
             # summary
             if cidx % cfg.val_summary_freq == 0:
+                v_outs = eval_outs[0] + [val_summary_op[0]]
```

Comment:
You can compute v_outs outside of the summary_freq if and then use v_outs[0] in the else branch at L 157